### DISCUSSION TITLE
Data design process

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#60] [General]
# Data design process

We want to make our data design process as open and transparent as possible.

We published [a blog explaining the stages of our process](https://digital-land.github.io/blog-post/our-data-standards-design-process/) and are currently working on a revised explanation of the stages.

We'd like to know your thoughts. Getting feedback from the community on what makes sense, what doesn't and where people think we have gaps would be very useful.

**Comment by psd** [comment]
My main thoughts are:

1. can we be clearer on the issues which need to be addressed for a consideration to move from one stage to the next?
2. is this process working? It feels very waterfall. I've noticed from local-plans that using real data changes design decisions. I think we should collapse the earlier stages into "discovery" and "co-design" to encourage more contributions earlier on:
    - discovery (for backlog, research, screen)
    - co-design (I guess this is alpha)
    - pilot (private beta?)
    - beta (on the platform at end of pilot)

4. this process ends with data on the platform. Are we closer to having stages for when a dataset needs to become a data standard using regulations or other duties placed on planning authorities and other parties to provide their data?

**Comment by psd** [comment]
Also, I can see how when a standard is set in law, like brownfield-land, the dataset becomes "live", but the consideration can still have further needs. Do we capture that in this consideration. It might be better to create new considerations for new or unmet needs? Something the process needs to consider.

**Reply by psd** [reply]
I think in this case, it's worth reviewing how we model users and needs, and clarify how a consideration can lead to many datasets.
